docs/documentation/components/table-expando-row.html

```diff
@@ -113,16 +116,16 @@ <h2>Benodigde stappen:</h2>
                 in de <code>&lt;head></code> van het document. Voor meer informatie zie:
                 <a href="../add-js.html">JavaScript toevoegen</a>
               </li>
+              <li>Open/sluit-icoon toevoegen:
+                <ul>
+                  <li>Voeg de gewenste icoon-classes toe via het HTML-attribuut <code>data-icon-open-class=""</code> voor het open-icoon en <code>data-icon-close-class=""</code> voor het sluit icoon. Bijvoorbeeld: <code>data-icon-open-class="icon chevron-down"</code>. Voor meer informatie zie het onderstaande html-voorbeeld.</li>
+                  <li>Om gebruik te maken van Manons eigen iconset gebruikt men de classes <code>"icon icon-descending"</code> en <code>"icon icon-ascending"</code> voor respectievelijk open- en sluit-iconen. </li>
```

Comment:
"Manons eigen iconset"? Leveren we ergens een voorbeeld-iconset mee? Want `manon-icons` gebruiken we dacht ik alleen "zelf" in `docs`.

Ik zie dat `icon-set-example.scss` inderdaad `.icon-ascending` en `.icon-descending` definieert, maar dat bestand dient zo te zien echt alleen als voorbeeld, en als de gebruiker ervoor kiest die zonder aanpassing over te nemen dan dient die zelf een passende `--icon-base-font-family` in te stellen met glyphs voor `\e901` en `\e902`.

Reply:
Ik heb de tekst geüpdatet met de correcte situatie, zoals door jou aangegeven. Tevens extra pointers naar de plek waar gebruikers zelf iconensets kunnen toevoegen.